### PR TITLE
Fix: Do not encourage to edit composer.json

### PR DESCRIPTION
**Pull request by localheinz**
This PR
- [x] updates `README.md` and suggests to require `league/plates` via command line, rather than editing `composer.json` directly

### CODE REVIEW

**Comment by reinink** [comment]
I agree with this, editing Composer was the only way to do it before, and this is a remnant from those days. I'm thinking all we need is this though:

```
composer require league/plates
```

If you want to update, cool, otherwise I can!

**Comment by localheinz** [comment]
@reinink 

Hang on a second!

File: README.md [hunk 1]
```diff
@@ -31,12 +31,8 @@ Plates is a native PHP template system that's fast, easy to use and easy to exte
...
 
-```json
-{
-    "require": {
-        "league/plates": "3.*"
-    }
-}
+```
+$ composer require league/plates
 ```
```
Comment (ragboyjr) [inline README.md]: Looks great, @reinink do you want the `$` or not? I just scanned various php league packages and some have the `$` and some don't.

- http://container.thephpleague.com/installation/
- http://commonmark.thephpleague.com/installation/

Comment (reinink) [inline README.md]: Probably not, but we can remove that post merge.

**Pull request merged**